src/pages/[platform]/build-a-backend/storage/upload-files/index.mdx

```diff
@@ -1202,15 +1202,50 @@ func getTempUrls(securityScopedUrls: [URL]) -> [URL] {
 
 <InlineFilter filters={["react", "angular", "javascript", "vue", "nextjs", "react-native"]}>
 
+### Upload to a specified bucket
+
+You can also perform an upload operation to a specific bucket by providing the `bucket` option. This option can either be a string representing the target bucket's assigned name in Amplify Backend or an object specifying the bucket name and region from the console.
+
+See [additional storage bucket usage](/[platform]/build-a-backend/storage/set-up-storage/#additional-storage-bucket-usage) for more information.
```

Comment:
Please remove this. The same info is available from the below table

Reply:
Removed. It might be worth keeping the link back to the main docs though?